plugins/node/opentelemetry-instrumentation-knex/package.json

```diff
@@ -33,7 +33,9 @@
   "files": [
     "build/src/**/*.js",
     "build/src/**/*.js.map",
-    "build/src/**/*.d.ts"
+    "build/src/**/*.d.ts",
+    "LICENSE",
+    "README.md"
```

Comment:
nit: This is fine, but I'd be inclined to not bother adding this. None of the other instrumentation packages in this repo have these. These two files are in the set of files that are always included by npm: https://docs.npmjs.com/cli/v11/configuring-npm/package-json#files

Reply:
removed, must have looked at another `package.json` in this repo for this probably